game/views.py

```
@@ -432,34 +446,43 @@ def offer_draw(request):
-        'valid': True,
-        'message': f'{resigning_player.capitalize()} resigned.',
-        'winner': winner,
-        'game_status': game_status
-    })
+        return JsonResponse({
+            'valid': True,
+            'message': f'{resigning_player.capitalize()} resigned.',
+            'winner': winner,
+            'game_status': game_status
+        })
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)